bubblecharts/charts/bubbleCloud.js

```diff
@@ -65,104 +65,104 @@ angular.module('myApp')
         // console.log(fontsize/scale)
           chart.attr("transform", "translate(" + d3.event.translate + ")scale(" + d3.event.scale + ")");
           chart.selectAll('.nodeTextToClip')
-          .style('font-size', function(d){ return d.r/scale/2.4})
-          .text(function(d){return clipText(d, d.name,fontsize/scale/2.5 );})
+          .style('font-size', function (d) { return d.r/scale/2.4})
+          .text(function (d) {return clipText(d, d.name,fontsize/scale/2.5 );})
         }
 
         var zoom = d3.behavior.zoom()
         .scaleExtent([-10, 50])
         .on("zoom", zoomed);
         zoom(svg);
-        
+
         function update(mentionFreq) {
             // console.log("updating");
             // console.log(mentionFreq);
             // if new poll, don't enter, fade out everything
             if(!mentionFreq || Object.keys(mentionFreq).length === 0) {
-                
+
                 chart.selectAll("text")
                 .transition().duration(1000)
                 .attr("opacity", 0)
-                
+
                 chart.selectAll("circle")
                 .transition().duration(1000)
                 .attr("r", 0)
-                
-                window.setTimeout(function(){
+
+                window.setTimeout(function () {
                     // select all bubble groups, remove data then remove dom nodes
                     chart.selectAll('.bubble').data([])
                     .exit().remove();
                 },1001);
-                
+
                 return;
             }
             // if ($scope.mentionFreq) chart.select("#loading").remove();
-            
-            
+
+
             function processData(obj) {
                 var newDataSet = [];
                 for(var prop in obj) {
-                    if(obj[prop] >= $scope.min){
+                    if(obj[prop] >= $scope.min) {
                         var node = {name: prop, className: 'bubbles', size: obj[prop]}
                         newDataSet.push(node);
                     }  
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/bubblecharts/charts/bubbleCloud.js
+++ b/bubblecharts/charts/bubbleCloud.js
@@ -105,7 +105,7 @@
                     if(obj[prop] >= $scope.min) {
                         var node = {name: prop, className: 'bubbles', size: obj[prop]}
                         newDataSet.push(node);
-                    }  
+                    }
                 }
                 return {children: newDataSet};
             }
```